src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
```

Comment:
Couldn't we include the location package and also include an attribute of type Location in our Event class?

Reply:
I thinking about doing that, but I left it like that so I could ask you guys next meeting what format you guys wanted the response? and go from there.